lib/OpenQA/CacheService/Client.pm

```diff
@@ -25,10 +25,12 @@ use OpenQA::Utils qw(base_host service_port);
 use Mojo::URL;
 use Mojo::File 'path';
 
-has attempts   => 3;
+# Define sensible defaults to cover even a restarting openQA webUI host being
+# down for up to 10m
+has attempts   => 120;
```

Comment:
What does 120 attempts mean? It doesn't sound "sensible" to re-try 120 times

Reply:
What else can you do to wait for the web UI becoming available again except polling? If you multiply it with the delay between the attempts you'll get 10 minutes like the comment states.

Reply:
Yes, the comment sounds explain it. I just don't like to repeat the values in comments to prevent a mismatch if someone changes the values but not updates the comment

Reply:
I'm not saying it's the wrong solution, just the comment doesn't tell me much. Is the database so slow that the web UI isn't ready for 10 minutes? Or the reverse proxy? Or something else?

Reply:
We sometimes have network issues, need to reboot the web UI host or have similar minor outages. The idea is simply not to produce tons of incompletes if the web UI is "shortly" offline.